docs/src/app/components/pages/components/CircularProgress/ExampleSimple.jsx

```diff
@@ -3,9 +3,9 @@ import CircularProgress from 'material-ui/lib/circular-progress';
 
 const CircularProgressExampleSimple = () => (
   <div>
-    <CircularProgress mode="indeterminate" />
-    <CircularProgress mode="indeterminate" size={1.5}/>
-    <CircularProgress mode="indeterminate" size={2}/>
+    <CircularProgress />
+    <CircularProgress size={1.5} />
+    <CircularProgress mode="indeterminate" size={2} />
```

Comment:
Sure. I assumed it was in the example by design, left one instance. I agree it's cleaner without, given that it's unneeded.